host/constants.py

```diff
@@ -31,7 +32,7 @@
     BASE['red_sphero'] = Point(1392, 114, 0)
     BASE['blue_sphero'] = Point(514, 981, 0)
 
-    ARENA_WIDTH_PIXELS = 1416 - 498  # assumed square
+    ARENA_WIDTH_PIXELS = 1027 - 212  # assumed square
```

Comment:
I think at some point we need to add better comments on each of these fields so others can better understand what they might need to change and if so how.  You've also got other things in constants that user's shouldn't change; i wonder if it makes sense doing 2 files - one for user-editable items and one not.  Regardless, too short need right now.